project-base/tests/ShopBundle/Smoke/NewProductTest.php

```diff
@@ -66,12 +66,21 @@ private function fillForm(Form $form)
         $form['product_form[basicInformationGroup][ean]'] = '123456';
         $form['product_form[descriptionsGroup][descriptions][1]'] = 'test description';
         $form['product_form[pricesGroup][productCalculatedPricesGroup][price]'] = '10000';
-        $form['product_form[pricesGroup][vat]']->select($this->getReference(VatDataFixture::VAT_ZERO)->getId());
+
```

Comment:
These blank lines visually chunk the form into blocks, but the block do not have any meaning...

Not necessary to change, but imo it would be more readable if you'd first get the necessary objects at the beginning of the method's body, then set the form values.

Reply:
I'm not sure how I should change it.